clouds.yaml.sample

```diff
@@ -0,0 +1,14 @@
+clouds:
+  openstack:
+    auth:
+      auth_url: "https://api.in-a-box.cloud:5000/v3"
+      username: "test"
+      password: "test"
+      project_name: "test"
+      project_domain_name: "Default"
+      user_domain_name: "Default"
+    region_name: "RegionOne"
+    interface: "public"
+    identity_api_version: 3
+    verify: false  # Skip TLS verification for self-signed certs
```

Comment:
Hmm, let's comment this one out and rather suggest to fill in a valid ca-bundle.crt, hinting that it would be the last cert in /etc/ssl/certs/ca-certificates.crt on the system with self-signed certs typically.
`verify: false` is an insecure workaround for cases where you lack it.